recipes/singularity-permanent-cache/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "singularity-permanent-cache" %}
+{% set version = "1.0.0a0" %}
```

Comment:
Please remove a0

Reply:
This is an alpha version.

Reply:
Is there a fully released version we can package. We tend not to package alpha versions, or other prereleases. There is a CFEP that you can check for instructions on how to do that, though

Reply:
@scopatz. Thanks! We will encourage users to install via PYPI and will open this PR to release 1.0.0 as soon as this RC proves to be stable.